packages/frontend/src/features/metricsCatalog/components/MetricsVisualizationEmptyState.tsx

```diff
@@ -0,0 +1,41 @@
+import { Center, Paper, Stack, Text } from '@mantine/core';
+import { IconClockCancel } from '@tabler/icons-react';
+import MantineIcon from '../../../components/common/MantineIcon';
+import MetricsVisualizationEmptyStateImage from '../../../svgs/metricsCatalog/metrics-visualization-empty-state.svg';
+
+export const MetricsVisualizationEmptyState = () => {
+    return (
+        <Paper
```

Comment:
does it have composition with `component` prop? if so you can probably do it with <Paper component={Flex} or {Center}

Reply:
Ah, I tried this but it still requires a few more props even after composition, so keeping it like this 👍